teamcity/resource_build_trigger_schedule.go

```diff
@@ -65,6 +66,44 @@ func resourceBuildTriggerSchedule() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 				Optional: true,
 			},
+			"cron_schedule": {
```

Comment:
As with the other schedule-related parameters, should this also have `ForceNew: true`?

Reply:
Yup, changed it to do just that.